website/pages/blog/scaling-out-elt-with-cq-and-csv.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Scaling out ELT pipelines with CloudQuery and CSV
+tag: tutorial
+date: 2022/10/04
+description: >-
+  This turoial will show you how to scale out your ELT pipelines with CloudQuery to infinity and beyond!
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Performance is a key factor in data pipelines both from usability and cost perspective. This tutorial will show you how to scale out you ELT pipelines with CloudQuery and CSV utilizing CloudQuery statless and performant architecture.
+
+## Background
+
+Let's take a look at your typical ELT pipeline:
+
+```
+[Source] -> [Database]
+```
+
+This is the basic architecture where you have one machine/container running your ELT workload scaling vertically and one database machine that needs also to scale vertically to handle the stream/load. This is a very common architecture and it works well for small to medium workloads. However, as your workload grows you will need to scale out your ELT workload. This is where things get complicated and expensive.
```

Comment:
database `instance` (instead of "machine"). Also too many "you"s, maybe say "However, as **the** workload grows you will need ... "